next.config.js

```diff
@@ -7,7 +7,11 @@ const withSourceMaps = require('@zeit/next-source-maps')
 
 const camelCaseLoader = path.join(__dirname, 'webpack/camelcase-loader.js')
 
+const envConfig = require('./config')
+
 const nextConfig = {
+  env: envConfig,
```

Comment:
Would be nice to include [these variables](https://github.com/oacore/dashboard/blob/master/next.config.js#L127-L132) in config too. (We can do it in another PR if you want)

Reply:
I will check if it's not passed implicitly. If it's not we should work more on this

Reply:
I feel okay with current configuration. Do you mind to keep it?

Reply:
I think all config variables should be in one place for the sake of simplicity. I don't insist on doing it now. It can be done in separate PR and I can do it as the code is closer to me more.

Reply:
Let's do to separately